templates/docker-compose/docker-compose.yml

```diff
@@ -31,7 +31,6 @@ services:
         environment:
             REDIS_HOST: ${CACHE_SERVICE_NAME}
             REDIS_PORT: ${REDIS_PORT}
-            HOST: ${HOST}
```

Comment:
The `HOST` environment var doesn't appear to be used anywhere now?

Reply:
THe `HOST` environment var is still there it's just moved position in the `docker-compose.yml` file. It's now under `env_file` and imported via the `.env.custom` file